plugins/azure-devops-backend/api-report.md

```diff
@@ -56,7 +56,7 @@ export class AzureDevOpsApi {
     },
   ): AzureDevOpsApi;
   // (undocumented)
-  getAllTeams(): Promise<Team[]>;
+  getAllTeams(teamsLimit?: number): Promise<Team[]>;
```

Comment:
I would suggest passing an object instead, as it makes it easy to evolve the method in the future.

```suggestion
  getAllTeams(options: { limit?: number }): Promise<Team[]>;
```

Reply:
@vinzscam  Does this apply only to the backend, only to this function, or in some broader scope? for example PullRequestsDashBoardProvider.readTeams()  which in turn calls azureDevOpsApi.getAllTeams()?

I can get it to work to pass an object around but since I don't know how that is useful I don't really know if its expected to be used all the way back in the pull request dashboard front end?